app/scripts/modules/core/src/application/config/footer/ConfigSectionFooter.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import autoBindMethods from 'class-autobind-decorator';
+
+export interface IConfigSectionFooterProps {
+  isDirty: boolean;
+  isValid: boolean;
+  isSaving: boolean;
+  saveError: boolean;
+  onRevertClicked: () => void;
+  onSaveClicked: () => void;
+}
+
+@autoBindMethods
+export class ConfigSectionFooter extends React.Component<IConfigSectionFooterProps> {
+  public render() {
+    const { isValid, isDirty, isSaving, saveError, onRevertClicked, onSaveClicked } = this.props;
+
+    if (!isDirty) {
+      return (
+        <div className="row footer">
+          <div className="col-md-3" />
+          <div className="col-md-9 text-right">
```

Comment:
just `col-md-12` and omit the div above?

Reply:
seems legit